charts/telegraf-operator/templates/certificate.yml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.certManager.certificate.enable -}}
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: {{ include "telegraf-operator.fullname" . }}
+  labels:
+    {{- include "telegraf-operator.labels" . | nindent 4 }}
+  namespace: {{ .Values.namespace }}
+spec:
+  dnsNames:
+  - "{{ include "telegraf-operator.fullname" . }}.{{ .Values.namespace }}.svc"
+  isCA: true
+  issuerRef:
+    kind: Issuer
+    name: selfsigned
+    namespace: {{ .Values.namespace }}
+  secretName: webhook-server-cert
```

Comment:
Should this be something like `telegraf-operator-tls` ?

Reply:
done